tests/simulation/translator/test_output_translation.py

```diff
@@ -216,141 +208,43 @@ def avg_surface_output_config_with_global_setting():
 
 
 def test_surface_ouput(
-    surface_output_config_with_global_setting,
-    surface_output_config_with_no_global_setting,
-    avg_surface_output_config_with_global_setting,
+    surface_output_config,
+    avg_surface_output_config,
 ):
-    import json
-
-    ##:: surfaceOutput with global settings
-    with SI_unit_system:
-        param = SimulationParams(outputs=surface_output_config_with_global_setting[0])
-    translated = {
-        "boundaries": {
-            "Wall1": {"type": "NoSlipWall"},
-            "Wall2": {"type": "NoSlipWall"},
-            "surface1": {"type": "NoSlipWall"},
-            "ZoneName/surface11": {"type": "NoSlipWall"},
-            "surface2": {"type": "NoSlipWall"},
-            "surface22": {"type": "NoSlipWall"},
-        }
-    }
-    translated = translate_output(param, translated)
-    assert sorted(surface_output_config_with_global_setting[1].items()) == sorted(
-        translated["surfaceOutput"].items()
-    )
-
     ##:: surfaceOutput with No global settings
     with SI_unit_system:
-        param = SimulationParams(outputs=surface_output_config_with_no_global_setting[0])
+        param = SimulationParams(outputs=surface_output_config[0])
     translated = {"boundaries": {}}
     translated = translate_output(param, translated)
-    assert sorted(surface_output_config_with_no_global_setting[1].items()) == sorted(
-        translated["surfaceOutput"].items()
-    )
+    assert sorted(surface_output_config[1].items()) == sorted(translated["surfaceOutput"].items())
 
     ##:: timeAverageSurfaceOutput and surfaceOutput
     with SI_unit_system:
-        param = SimulationParams(
-            outputs=surface_output_config_with_no_global_setting[0]
-            + avg_surface_output_config_with_global_setting
-        )
+        param = SimulationParams(outputs=surface_output_config[0] + avg_surface_output_config)
     translated = {"boundaries": {}}
     translated = translate_output(param, translated)
     ref = {
         "animationFrequency": 123,
         "animationFrequencyOffset": 321,
-        "animationFrequencyTimeAverage": 23,
-        "animationFrequencyTimeAverageOffset": 21,
+        "animationFrequencyTimeAverage": -1,
```

Comment:
Why are the reference value changed? Because these were previously "global setting"?

Reply:
Yeah, they were previous the "global" values.